remodel/models.py

```diff
@@ -76,12 +76,13 @@ def save(self):
             result = (r.table(self._table).get(id_).replace(r.row
                         .without(r.row.keys().difference(list(fields_dict.keys())))
                         .merge(fields_dict), return_changes='always').run())
-
-        except KeyError:
+            if result['errors'] > 0:
```

Comment:
On a side note - i'm looking back at this code...  The KeyError would be raised in line 75.  I'm breaking things because my error is coming from the next line: from the query itself.